docs/site/Troubleshooting.md

```diff
@@ -0,0 +1,92 @@
+---
+lang: en
+title: 'Troubleshooting'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, Debug
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Troubleshooting.html
+---
+
+In general, when you are developing an application, use the `npm` command to run
+it. This enables you to see stack traces and console output immediately.
+
+For example:
+
+```
+$ cd myapp
+$ npm start
+```
+
+LoopBack 4 also provides multiple ways to help you verify or diagnose the app.
+
+## Setting Debug Strings
+
+If you get an error but it doesn't provide enough information, you can turn on
```

Comment:
Since we have a dedicated page for debug strings, perhaps we can simply the section by:

```
If you get an error but it doesn't provide enough information, you can turn on debug mode by starting your application using `DEBUG=<DEBUG_STRING> npm start`. [Setting debug strings](Setting-debug-strings.md) for details.
```